src/local-editor-service.ts

```diff
@@ -98,21 +99,55 @@ export class LocalEditorService extends EditorService {
           };
         })
       };
-    } else if (location.kind === 'attribute') {
+    }
+
+    if (location.kind === 'attributeValue') {
+      const domModule =
+          this.getAncestorDomModuleForElement(document, location.element);
+      if (!domModule || !domModule.id)
+        return;
+      const outerElement = document.getOnlyAtId('element', domModule.id);
+      if (!outerElement)
+        return;
+      const sortPrefixes = this._createSortPrefixes(outerElement);
+      const innerElement =
+          document.getOnlyAtId('element', location.element.nodeName)!;
+      const innerAttribute = innerElement.attributes.find(
```

Comment:
This will throw if the inner element is unknown to the analyzer. Probably better to remove the `!` on the preceding line and add another `if (...) return;`